src/sdg/conf/base.py

```diff
@@ -466,6 +464,17 @@
 
 ACCOUNT_ADAPTER = "sdg.accounts.adapters.AccountAdapter"
 
+# Markdown
+MARKDOWNIFY_WHITELIST_TAGS = [
+    "table",
+    "thead",
+    "tbody",
+    "th",
+    "tr",
+    "td",
+]
```

Comment:
I'm not sure how this works, but we should NOT allow HTML-tags in markdown.

I think we should actually have a validator for those model fields that check for HTML-tags and if present it should give an error that bubbles up in the user interface.

Reply:
This whitelist is needed to render the markdown tables to html.